src/extensions/replay/config.ts

```diff
@@ -60,10 +64,24 @@ const HEADER_DENYLIST = [
     'x-xsrf-token',
 ]
 
+const PAYLOAD_CONTENT_DENY_LIST = [
+    'password',
+    'secret',
+    'passwd',
+    'api_key',
+    'apikey',
+    'auth',
+    'credentials',
+    'mysql_pwd',
+    'privatekey',
+    'private_key',
+    'tokenconst',
+]
+
 // we always remove headers on the deny list because we never want to capture this sensitive data
 const removeAuthorizationHeader = (data: CapturedNetworkRequest): CapturedNetworkRequest => {
-    Object.keys(data.requestHeaders ?? {}).forEach((header) => {
-        if (HEADER_DENYLIST.includes(header.toLowerCase())) delete data.requestHeaders?.[header]
+    _each(Object.keys(data.requestHeaders ?? {}), (header) => {
```

Comment:
Random one - I don't like the `_each` function - we should eventually make it explicit like `_iterateArray` and `_iterateObject` as currently it screws the typing